plugins/do-kvm.py

```diff
@@ -64,11 +64,17 @@ def remove_mounts():
 
 
 def is_passable_mount(v):
```

Comment:
On HPCAC, this was needed to get the rdma-core directory passed through:
```
mkt run --dir /images/ztiffany/src/rdma-core/
```
Is this expected?

Reply:
no, it means your config file is incomplete or another bug, we mount whole src directory

Reply:
It could be because /images is on tmpfs, but I don’t think we saw even an attempt to mount it